ftest.sh

```diff
@@ -188,7 +212,7 @@ shift || true
 args+=" $*"
 
 # shellcheck disable=SC2029
-if ! ssh -i ci_key jenkins@"${nodes[0]}" "set -ex
+if ! ssh "${REMOTE_ACCT:-jenkins}"@"${nodes[0]}" "set -ex
```

Comment:
(lint) The double quotes around this do nothing. Remove or escape them. [SC2140]